File: ToyInterpreter 2/src/mobserver/OutBufferObserver.java
package mobserver;

import Model.IOutBuffer;

public class OutBufferObserver extends CObserver implements java.io.Serializable {
	@Override
	public void onUpdate(Object o) {
		System.out.println("Out Buffer:\n");
		System.out.println(o.toString());
	}
}

File: ToyInterpreter/ToyInterpreter/Model/MObserver/OutBufferObserver.cs
using System;

namespace ToyInterpreter.Model.MObserver
{
	[Serializable()]
	public class OutBufferObserver : CObserver
	{
		public override void onUpdate (Object o)
		{
			Console.WriteLine ("OutBuffer:");
			Console.WriteLine (o.ToString());
		}
	}
}

